src/DotLiquid/SyntaxCompatibilityEnum.cs

```diff
@@ -0,0 +1,18 @@
+namespace DotLiquid
+{
+    /// <summary>
+    /// Strict Liquid Syntax Compatibility Flag
+    /// </summary>
+    public enum SyntaxCompatibility
+    {
+        /// <summary>
+        /// Behavior as of DotLiquid 2.0
+        /// </summary>
+        COMPAT_2_0 = 200,
```

Comment:
Version2_0 (or Version_2_0)

In order to:
* Avoid all-caps
* Avoid non-standard abbreviation
* Substitute _ for . but not for space characters

Reply:
You are right about case. Compat is definitely a used abbreviation (see referencesource.microsoft.com) but Version is a nice alternative. As far as underscore it seems the standard is to omit completely so Tls13 is 1.3.so it will be Version20 and Version21. Only Q is should we worry about Version 2.10 or will we bump to 3.x by then?